### PR TITLE
Set UTF-8 as default encoding for peep

### PR DESCRIPTION
**Pull request by MikkCZ**
This was necessary for me to make the build, each time I used peep, work. Maybe will help someone too, and hopefully not break anything.

### CODE REVIEW

**Comment by edmorley** [comment]
Thank you for the PR, unfortunately it's failing on Python 3.

**Comment by MikkCZ** [comment]
I found that for Python 3 the default encoding is already UTF-8 and even cannot be changed. r?

**Comment by edmorley** [comment]
I defer to @erikrose for review :-)

**Comment by MikkCZ** [comment]
@erikrose r?

File: peep.py [hunk 1]
```diff
@@ -23,6 +23,14 @@
 # all copies or substantial portions of the Software.
 from __future__ import print_function
 try:
+    reload
+    import sys
+    reload(sys)
```
Comment (erikrose) [inline peep.py:28]: reload() scares me for various reasons. (For example, symbols defined in the previously loaded incarnation of the module will be duplicated, leading to fun behaviors like previously created instances of classes starting to fail isinstance() checks.) Can you explain why reload() is necessary here and whether there are any alternatives?

**Comment by MikkCZ** [comment]
Sorry, I stopped working with peep since the times and created the patch. Feel free to close this PR.

**Comment by erikrose** [comment]
Thanks. Peep being deprecated anyway, I think we'll close this until somebody else stumbles upon a similar problem.

**Pull request closed without merging**